libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.ts

```diff
@@ -35,7 +37,11 @@ export class WattNavListComponent {}
 
 @NgModule({
   declarations: [WattNavListComponent],
-  exports: [WattNavListComponent, WattNavListItemScam],
+  exports: [
+    WattNavListComponent,
```

Comment:
issue: Exporting a component doesn't make sense until Angular supports Standalone Components. Remove this export. This can be done in a separate PR if preferred.
```suggestion
```